assets/src/components/searchForm.tsx

```diff
@@ -54,7 +54,82 @@ type SearchFormProps = SearchFormEventProps & {
   /**
    * Callback to run when {@link filters} should be updated.
    */
-  onFiltersChanged: (name: SearchProperty, currentValue: boolean) => void
+  onFiltersChanged: (searchFilterState: SearchFiltersState) => void
+}
+
+const allFiltersOn: SearchFiltersState = {
+  vehicle: true,
+  operator: true,
+  run: true,
+  location: true,
+}
+const allFiltersOff: SearchFiltersState = {
+  vehicle: false,
+  operator: false,
+  run: false,
+  location: false,
+}
+
+const Filters = ({
+  filters,
+  onFiltersChanged,
+}: {
+  filters: SearchFiltersState
+  onFiltersChanged: (searchFilterState: SearchFiltersState) => void
+}) => {
+  const countActiveFilters = Object.values(filters).filter(
+    (isActive) => isActive
+  ).length
+  const [hasAManuallyActivatedFilter, setHasAManuallyActivatedFilter] =
+    useState(
+      countActiveFilters > 0 && countActiveFilters < Object.keys(filters).length
+    )
+
+  const allFiltersActive = Object.values(filters).every((isActive) => isActive)
+  const displayAsInactive = allFiltersActive && !hasAManuallyActivatedFilter
+  return (
+    <FilterAccordion.WithExpansionState heading="Filter results">
+      {Object.entries(filters)
+        .map(([property, isActive]) => ({
+          property: property as SearchProperty,
+          isActive,
+        }))
+        .sort(
+          ({ property: first_property }, { property: second_property }) =>
+            searchPropertyDisplayConfig[first_property].order -
+            searchPropertyDisplayConfig[second_property].order
+        )
```

Comment:
**suggestion (non-blocking):** since we sort `SearchProperty`s by order in a few places maybe it would be worth making this a helper function.